app/src/main/res/layout/activity_login.xml

```diff
@@ -69,75 +91,121 @@
     </LinearLayout>
 
     <TextView
+        android:id="@+id/textView"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_alignTop="@+id/upper_poly"
         android:layout_alignEnd="@+id/upper_poly"
         android:layout_marginStart="230dp"
-        android:layout_marginTop="420dp"
+        android:layout_marginTop="160dp"
+        android:layout_marginBottom="2dp"
         android:text="Forgot?"
```

Comment:
'Forgot Password?' string will be in strings.xml

Reply:
Done